test/documentation/version.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/* Jongkyu Kim(j.kim@fu-berlin.de), 2016.01.12 */
+$LOCALDIR = "../";
+
+$files = scandir($LOCALDIR, SCANDIR_SORT_DESCENDING);
+$list = array();
+foreach( $files as $file  )
+{
+    if( strpos($file, "master_") !== FALSE )
+        continue;
+    if( $file == "master")
+        continue;
+    if( strpos($file, "develop") !== FALSE )
+        continue;
+    if( $file[0] == "1")
```

Comment:
This is done to skip seqan versions 1 and 2 right? Add a comment either way.
Do we really want to also skip learning_resources?

Reply:
Mhm, I don't know if we really need it in the API docs. the learning resources don't necessarily have something to do with SeqAn....

Also, it's rather easy to change afterwards by just replacing version.php on the server.

Reply:
Also, what is the order of the enries we would like?

Currently
```
3.0.1
3.0.0
3-master-user
3-master-dev
```

Reply:
I agree, that we don't want to display the learning resources in the SeqAn API documentation. The labels are based on the directories right? Some general thoughts: Since we have only one branch called master we don't need to call the documentation master. It should be rather `head` which reflects more that it is based on the latest commit. Second, the choices for user and dev bad I think. I propose to change the general url to:
```
http://docs.seqan.de/seqan3/library_user_head/
http://docs.seqan.de/seqan3/library_developer_head/
```
And accordingly change the old seqan link to seqan2 in the top level. That must be done however separately and maybe before we tackle the 2.4.1 release.

Reply:
I agree and would do another PR for the directory changes after this one is merged?

Reply:
yes, in a separate PR would be great.